packages/components-native/src/Disclosure/__snapshots__/Disclosure.test.tsx.snap

```diff
@@ -370,7 +370,7 @@ exports[`should not render the caret when the Disclosure is empty 1`] = `
     }
     accessible={true}
     collapsable={false}
-    focusable={true}
+    focusable={false}
```

Comment:
this change is expected

<img width="720" alt="image" src="https://github.com/user-attachments/assets/7f50be93-250e-49a8-9eea-1958ca1698d6" />

where in TouchableOpacity prior to `0.75.0` the logic for `focusable` was 

```
        focusable={
          this.props.focusable !== false && this.props.onPress !== undefined
        }
```

and in `0.75.0`

```
        focusable={
          this.props.focusable !== false &&
          this.props.onPress !== undefined &&
          !this.props.disabled
        }
```

and our `TouchableOpacity` in this snapshot is disabled

```
    <TouchableOpacity
      activeOpacity={tokens["opacity-pressed"]}
      onPress={onToggle}
      disabled={isEmpty}
    >
```